### DISCUSSION TITLE
GitHub Actions does not respect [no ci]

### DISCUSSION THREAD

**Discussion by wdzeng** [community/community#26677] [Actions]
<p>It is noted that if a commit message contains <code>[no ci]</code>, the workflow is omitted in <a href="https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs" rel="noopener nofollow ugc">this article</a>. However a workflow was still triggered in <a href="https://github.com/wdzeng/The-Content-Farm-Terminator/actions/runs/2297778388" rel="noopener nofollow ugc">this case</a>. Did I do anything wrong?</p>
<p>By the way, if I replace <code>[no ci]</code> with <code>[skip ci]</code> in commit message, then the workflow is skipped as expected.</p>

**Comment by jsoref** [comment, marked as answer]
<p>You wrote <code>[no-ci]</code>, but that <code>-</code> meant it didn’t match <code>[no ci]</code></p>

**Comment by wdzeng** [comment]
<p>My bad. I am sorry for this stupid post.</p>
